### DISCUSSION TITLE
新版本中间结果 content_list.json 为什么删除了

### DISCUSSION THREAD

**Discussion by Tendo33** [magicpdf/Magic-PDF#393] [General]
对清洗数据来说，content_list.json应该是最好的中间结果了吧

**Comment by myhloli** [comment]
content list格式准备升级，在重构cli的时候就先去掉了，考虑到还有用户需要使用contentlist，在下个版本会增加一个debug开关来临时输出contentlist

**Comment by myhloli** [comment]
可以在命令行加-d true来通过debug模式输出contentlist了

**Reply by Tendo33** [reply]
万岁！
